notes/points-markets-settlement.md

```diff
@@ -0,0 +1,103 @@
+
+Note: adjustments to points balances such as normal or reverse splits (1 point becomes, 2 points, 10 points, 100 points, 0.5 points, 0.1 points, etc.), multiplying some users points by some value, etc. etc. do not constitute an *Airdrop Event* and simply impact the market price of a point as an issuance of new points would.
+
+Once an ***Airdrop Event*** has occurred and all input to the ***Valuation Algorithm*** is available, the *mean conversion rate* is calculated and the market will be settled. 
+
+
+### Imminent Airdrop Event
+
+An ***Imminent Airdrop Event*** will occur following any official confirmation (including implied confirmation as a result of the action happening before any official communication) of a future *conversion*, listing, or other action that would constitute an ***Airdrop Event***. 
+
+For an announcement or action to trigger an ***Imminent Airdrop Event*** it must originate from or be endorsed as truthful and accurate by the project or another entity with sufficient control to initiate an ***Airdrop Event***.
+
+The *termination date/time* shall be `Early Termination Period` before any announcement in which the specifics of the ***Airdrop Event*** that are required by the ***Valuation Algorithm*** are confirmed.
+
+In the event that the data required by the ***Valuation Algorithm*** are available with less than `Early Termination Period` of notice (either due to an unannounced ***Airdrop Event*** or listing, or due to a short period between announcement and implementation), the *termination date/time* will be the current date/time (i.e. termination is immediate).
+
+Note that due to the specifics of oracle protocols, etc. there may be a delay between the conditions for triggering an ***Imminent Airdrop Event*** and the data reaching a market on-chain and the cessation of trading.
+
+
+### Valuation Algorithm
+
+Once an airdrop event has been declared, it is necessary to calculate a value per point in order to settle the market. In some cases this may be possible immediately, and in others more information may be required even once the Airdrop Event has been triggered.
+
+In cases where points have undergone *conversion* as defined above, or where *conversion* is no longer possible (***Airdrop Event*** cases 1 & 2):
+
+- The *mean conversion rate* is the total value in the `Base Asset` distributed to *points* holders (which may be zero in some cases) divided by the total number of *points* that have either undergone *conversion* or been destroyed / become valueless.
+
+
+In the case where *points* become tradable (***Airdrop Event*** case 3):
+
+- The *mean conversion rate* is the volume weighted average price (VWAP) of one point (converted to the `Base Asset` is necessary) over `Minimum Trading Window` period prior to the ***Airdrop Event***.
+
```

Comment:
Also, may be worth specifying rounding methodology such that the number is exactly equivalent to what the Vega market would want (and also to avoid any arguments when voting that the number should have more/less precision)

Reply:
Can we suggest some wording around this into an appendix to avoid any challenges on technicalities in Uma

Reply:
Added to WIP: appendix